### PR TITLE
refactor(components): Update typography for Button

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations
Match new style for Button component because the typography has been updated

## Changes
Before:
image

After:
image

### Changed
- updated casing, spacing and font-family of typography within the button
- snapshot tests that use button have been updated

### Testing
- check Button component locally and compare with Figma
- can check casing by updating playground
---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Button/Button.css [hunk 1]
```diff
@@ -16,22 +16,24 @@
   border: var(--border-base) solid var(--button--color-variation);
   border-radius: var(--radius-base);
   color: var(--button--color-variation);
-  fill: var(--button--color-variation);
   -webkit-font-smoothing: antialiased;
+  letter-spacing: 0.2px;
```
Comment (chris-at-jobber) [inline packages/components/src/Button/Button.css:20]: We should use the value from the [design package](https://github.com/GetJobber/atlantis/blob/15f604cc51ee6881a5ed9c1be01081c76cf5ce6e/packages/design/src/typography.css#L3) here
```suggestion
  letter-spacing: var(--typography--letterSpacing-loose);
```

File: packages/components/src/Button/Button.css [hunk 2]
```diff
@@ -16,22 +16,24 @@
...
   text-align: center;
   text-decoration: none;
   vertical-align: middle;
   background-color: var(--button--color-surface);
   cursor: pointer;
   transition: all var(--timing-base);
   appearance: none;
+  fill: var(--button--color-variation);
```
Comment (chris-at-jobber) [inline packages/components/src/Button/Button.css:28]: Did this move itself on save (auto-formattting?), or was this a specific decision?

File: packages/components/src/Button/Button.tsx [hunk 3]
```diff
@@ -162,8 +162,9 @@ function ButtonInternals({ label, icon, size = "base" }: ButtonProps) {
       {icon && <Icon name={icon} size={size} />}
       <Typography
         element="span"
-        textCase="uppercase"
+        textCase="capitalize"
```
Comment (chris-at-jobber) [inline packages/components/src/Button/Button.tsx:165]: I know I wrote the ticket but thinking about we shouldn't actually use `capitalize` here. I'd like to follow the same guidelines we have for [mobile Button](https://atlantis-mobile.jobber.dev/?path=/docs/atlantis-actions-button--button) around "proper" title-casing – it does add more work on the implementation-in-product side as we'll have a bunch of labels to tweak, but we should be able to catch those instances fairly easily.

**Review by chris-at-jobber — CHANGES_REQUESTED** [review]
One change to a variable used, and one larger change!

File: packages/components/src/Button/Button.css (hunk 2 above)
Comment (taylorvnoj) [reply packages/components/src/Button/Button.css:28]: @chris-at-jobber yes, this moved itself. I noticed there's quite a few things my vsCode changes which go against the Atlantis linter, so I'll be making sure those settings are adjusted

File: packages/components/src/Button/Button.mdx [hunk 4]
```diff
@@ -129,9 +129,9 @@ Destructive actions remove something from Jobber.
 #### Primary
 
 A Primary destructive action will permanently destroy an object carrying
-signficant data, such as a job, quote, or client. This destructive action should
-be contained in a [ConfirmationModal](/components/confirmation-modal) triggered
-by clicking a secondary or tertiary destructive button.
+significant data, such as a job, quote, or client. This destructive action
```
Comment (taylorvnoj) [inline packages/components/src/Button/Button.mdx:132]: This was just to fix a typo (spelling of `significant`)

File: packages/components/src/Button/Button.tsx (hunk 3 above)
Comment (taylorvnoj) [reply packages/components/src/Button/Button.tsx]: Good call out - I removed the `textCase` variable and updated the `.mdx` file to match the guidelines we have for mobile.  Like we discussed, once we are updating the layouts in the future ticket, we will look at `<Button>` components in JO, Dev Center, etc. I don't mind doing that when the time comes 👍

**Review by chris-at-jobber — APPROVED** [review]
Looks great, thanks @taylorvnoj !

**Pull request merged**